.github/workflows/build-xcframework.yml

```diff
@@ -0,0 +1,75 @@
+name: Build XCFramework
+
+on:
+  pull_request:
+
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  build_and_assemble_xcframework:
+    name: Build And Assemble XCFramework
+    # run on mac so iOS compilation is executed as well
+    runs-on: macos-15
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
+      - name: Setup Java JDK
+        uses: actions/setup-java@v5
+        with:
+          distribution: "temurin"
+          java-version-file: ".java-version"
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v5
+      - name: Cache Konan
+        uses: actions/cache@v4
+        with:
+          path: ~/.konan
+          key: ${{ runner.os }}-konan-${{ github.sha }}
+          restore-keys: ${{ runner.os }}-konan-
+      - name: Build
+        run: ./gradlew assemble --scan
+      - name: Run Command Gradle AssembleXCFramework
+        run: ./gradlew assembleGodToolsSharedXCFramework --warning-mode all
+  
+  build_zip_from_xcframework:
+    name: Build Zip From XCFramework
+    runs-on: macos-latest
+    needs: [ build_and_assemble_xcframework ]
+    steps:
+      - name: Run Command Zip
+        run: zip -r /Users/runner/work/kotlin-mpp-godtools-tool-parser/kotlin-mpp-godtools-tool-parser/build/XCFrameworks/release/GodToolsShared.xcframework.zip /Users/runner/work/kotlin-mpp-godtools-tool-parser/kotlin-mpp-godtools-tool-parser/build/XCFrameworks/release/GodToolsShared.xcframework
```

Comment:
each job has a clean environment when they start, so you won't have access to the build artifacts from the build framework job

Reply:
Thanks Daniel yep just realized that :D

Reply:
That's true, that way don't have to update 2 places